api/src/home/analysis-platform/job_handlers/local_container/__init__.py

```diff
@@ -64,10 +66,12 @@ def start(self) -> str:
                 detach=True,
             )
 
-        except KeyError as error:
+        except (KeyError, AttributeError) as error:
             raise Exception(
-                f"Job entity used as input to local container jobs does not include required attribute {error}"
+                f"Job entity used as input to local container jobs is missing: {error}. Please make required changes and create a new job."
```

Comment:
These are also generic enough that they could be handled by caller function. They don't add much information to the exception.